### ISSUE TITLE
port TLS_ENABLE from OvmfPkg to ArmVirtPkg (Bugzilla Bug 1009)

### ISSUE THREAD

**Issue by tianocore-issues** [tianocore/edk2#7362]
This issue was created automatically with [bugzilla2github](https://github.com/makubacki/bugzilla2github/tree/bz2gh_tianocore)
# Bugzilla Bug 1009
Date: 2018-07-12T07:05:37+00:00
From: @lersek
To:   @lersek
CC:   @ardbiesheuvel, f4bug, guillaume.gardet, @lersek

Depends on:   #1293
Last updated: 2019-06-28T12:13:58+00:00

**Comment by tianocore-issues** [comment]
## Comment 4272
Date: 2018-07-12 07:05:37 +0000
From: @lersek

- Industry Specification: ---
- Release Observed: EDK II Master
- Releases to Fix: EDK II Master
- Target OS: ---
- Bugzilla Assignee(s): @lersek

+++ This bug was initially created as a clone of Bug #1007 +++

The OvmfPkg DSC/FDF files support the NETWORK_IP6_ENABLE and TLS_ENABLE boolean build flags, for enabling the respective networking features.

Port them to the ArmVirtPkg DSC/FDF files.

---

Splitting the two features to separate BZs so we can handle them with finer granularity. This BZ covers TLS_ENABLE.

**Comment by tianocore-issues** [comment]
## Comment 7111
Date: 2019-04-12 08:59:41 +0000
From: @lersek

*** Bug 1714 has been marked as a duplicate of this bug. ***

**Comment by tianocore-issues** [comment]
## Comment 7112
Date: 2019-04-12 09:18:44 +0000
From: @lersek

I have had a patch ready for this ever since filing the bug, but I haven&apos;t posted it due to ... reasons I can&apos;t disclose just yet. I&apos;m asking for a bit more patience.

**Comment by tianocore-issues** [comment]
## Comment 7128
Date: 2019-04-15 02:50:45 +0000
From: Guillaume GARDET &lt;<guillaume.gardet>&gt;

(In reply to Laszlo Ersek from comment #2)
&gt; I have had a patch ready for this ever since filing the bug, but I haven&apos;t
&gt; posted it due to ... reasons I can&apos;t disclose just yet. I&apos;m asking for a bit
&gt; more patience.

Any ETA for your patch? Maybe you could share it, so that I can test it, please?

**Comment by tianocore-issues** [comment]
## Comment 7130
Date: 2019-04-15 07:51:41 +0000
From: @lersek

Guillaume, this issue is blocked by bug 960. Please speak with the TianoCore InfoSec group, regarding the ETA on bug 960. If ARM.com has someone delegated to the TianoCore InfoSec group, they can CC you on bug 960, and then you can read through it. (The latest comment there at present is #25.)

In addition, Ard can now provide you with some background info.

**Comment by tianocore-issues** [comment]
## Comment 8162
Date: 2019-06-24 11:31:25 +0000
From: @lersek

Removing the dependency on bug 960, due to bug 960 comment 37. (It&apos;s public knowledge now that HTTPS boot should not be trusted until bug 960 is fixed.)

**Comment by tianocore-issues** [comment]
## Comment 8166
Date: 2019-06-24 15:15:58 +0000
From: @lersek

Posted
[PATCH] ArmVirtPkg: handle NETWORK_TLS_ENABLE in ArmVirtQemu*
http://mid.mail-archive.com/20190624191336.31611-1-lersek@redhat.com
https://edk2.groups.io/g/devel/message/42754

**Comment by tianocore-issues** [comment]
## Comment 8279
Date: 2019-06-28 12:13:58 +0000
From: @lersek

Fixed in commit ffe048a0807b (&quot;ArmVirtPkg: handle NETWORK_TLS_ENABLE in ArmVirtQemu*&quot;, 2019-06-28).
